antarest/study/web/studies_blueprint.py

```diff
@@ -356,6 +356,7 @@ def copy_study(
         uuid: str,
         dest: str,
         with_outputs: bool = False,
+        output_ids: Annotated[list[str] | None, Query()] = None,
```

Comment:
Actually, for query parameters, we cannot distinguish between an empty list and a `None`. So it would probably be simpler to have a non optional and an empty list as the default here ?

Reply:
I set it this way to not have a mutable value as default. We don't want that

Reply:
You may use `Query(default_factory=list)` for that purpose (but indeed it's not that different)